modules/ROOT/pages/to-publish-assets-maven.adoc

```diff
@@ -21,21 +22,27 @@ operations on your computer or server.
 
 Include the required plugin `org.mule.tools.maven.mule-maven-plugin` in your POM file. Do not include the plugin `org.apache.maven.plugins.maven-deploy-plugin` in your POM file or settings file, because it is not compatible with the required plugin.
 
+Using Anypoint Exchange's Maven Facade API version 3 to publish assets to Exchange requires the Mule Maven plugin version 3.5.0 or later.
+
 Note: You must have a unique artifact name and organization ID to publish your asset.
 
 == View Your Organization ID
 
 To construct the elements of a Maven POM file, you need to determine your organization ID:
 
-. Log into Anypoint Platform and click Access Management.
-. Click the organization name for the top level or a business group, and view your Organization ID in the Organization Info screen.
-. Copy the organization ID and add it to the examples that follow for how to set up your project's POM file.
+. Log in to Anypoint Platform and click Access Management.
+. Click the organization name for the top level or a business group, and view your organization ID in the organization info screen.
```

Comment:
```suggestion
. Click the name of the top-level organization or a business group, and view your organization ID in the organization information screen.
```
"a business group," not necessarily top level, or just "business group," also top level?